crates/biome_cli/tests/snapshots/main_commands_migrate_eslint/migrate_eslintrc.snap

```diff
@@ -72,6 +72,23 @@ biome.json migrate ━━━━━━━━━━━━━━━━━━━━
       21 │ + 
   
 
+```
+
+```block
+migrate ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i 100% (6/6) of the rules can be migrated.
+  
+  i Rules that can be migrated:
+  
+  - default-param-last
+  - dot-notation
+  - eqeqeq
+  - getter-return
+  - no-eval
+  - no-extra-label
```

Comment:
Maybe it makes sense to make an exception if 100% can be migrated. In this case, I don't think the user is interested anymore in a list of the individual rule names, so we could leave it out.

Reply:
Good point. Maybe we could do the same when nothing is migrated.